emop-core/pom.xml

```diff
@@ -16,7 +16,7 @@
     <dependency>
       <groupId>edu.illinois</groupId>
       <artifactId>starts-core</artifactId>
-      <version>1.4-SNAPSHOT</version>
+      <version>1.6-SNAPSHOT</version>
```

Comment:
We never released emop on Maven central AFAIK, so why do you need to bump up the version?